library/src/containers/AsyncApi/AsyncApi.tsx

```diff
@@ -1,6 +1,9 @@
 import React, { Component } from 'react';
 import { Options as ParserOptions } from 'json-schema-ref-parser';
 
+// @ts-ignore
+import { register } from 'web-react-components';
```

Comment:
Could we create separate file, maybe `AsyncApiWebComponent` and wrap `AsyncApiComponent` component from this file to new one? I mean making something like:

```js
import React, { Component } from 'react';
// @ts-ignore
import { register } from 'web-react-components';
import AsyncApiComponent from './AsyncApi';

export class AsyncApiWebComponent extends Component<AsyncApiProps> {
   constructor(props) {
      super(props);
    }

  render() {
    <>
        <style>@import '/assets/async-api/fiori.css';</style>
        <AsyncApiComponent {...this.props} />
    </>
  }
}

register(AsyncApiWebComponent, 'async-api-component', ['schema', 'config']);
```

Why is it better? If someone use React, then want only have dependencies related to react, not unnecessary deps like deps for web-component. Also final bundle for react app is bigger, and this is not good.

I also think that web-component should have separate package, but you can leave it to us :)